webui/react/src/e2e/models/components/ExperimentActionDropdown.ts

```diff
@@ -1,9 +1,22 @@
-import { DropdownContent } from 'e2e/models/common/hew/Dropdown';
+import { BaseComponent } from 'e2e/models/common/base/BaseComponent';
+import { DropdownMenu } from 'e2e/models/common/hew/Dropdown';
+import { Modal } from 'e2e/models/common/hew/Modal';
 
 /**
  * Represents the ExperimentActionDropdown component in src/components/ExperimentActionDropdown.tsx
  */
-export class ExperimentActionDropdown extends DropdownContent {
-  // TODO where is this thing? <Button icon={<Icon name="overflow-vertical" size="small" title="Action menu" />} />
-  // TODO I'm assuming it's new tab, new window, copy value, etc
+
+class ExperimentEditModal extends Modal {
```

Comment:
can you please move this class to `components/ExperimentEditModal.ts`

Reply:
im asking because of this line
https://github.com/determined-ai/determined/blob/main/webui/react/src/components/ExperimentActionDropdown.tsx#L15